src/invoice2data/extract/parsers/lines.py

```diff
@@ -25,6 +25,10 @@ def parse_block(template, field, settings, content):
     # Validate settings
     assert "line" in settings, "Line regex missing"
 
+    logger.debug(
+        "START line block content ========================\n%s", content
+    )
+    logger.debug("END line block content ==========================")
```

Comment:
Can you replace "line" with "lines", please?

Reply:
:+1 Good catch!